splash-screen/android/src/main/java/com/capacitorjs/plugins/splashscreen/SplashScreenPlugin.java

```diff
@@ -18,7 +18,11 @@ public class SplashScreenPlugin extends Plugin {
     public void load() {
         config = getSplashScreenConfig();
         splashScreen = new SplashScreen(getContext(), config);
-        splashScreen.showOnLaunch(getActivity());
+        if (!bridge.isMinimumWebViewInstalled() && bridge.getConfig().getErrorPath() != null && !config.isLaunchAutoHide()) {
+            return;
```

Comment:
I don't really understand the question.

At the moment, if the user has configured an error page, and the WebView is older than supported and they are using `launchAutoHide: false`, the error page is still now shown because the Splash Screen is on top of it.
And the user can't programmatically call `SplashScreen.hide()` to hide it because as the WebView is older than supported, plugins don't work, and also the error page has no access to Capacitor plugins.
So what this PR does is to detect those 3 problematic scenarios and if the 3 of them are happening, it doesn't show the Splash Screen so it shows the error page directly.

So, if the WebView is new, there is no problem, it will show the splash and then the call to `SplashScreen.hide()` will work.
If the WebView is old, but there is no configured error page, nothing we can do in this case as the user is not handling unsupported WebViews. But Capacitor shows an error on logcat for users to see.
If the WebView is old and the error page is configured but `launchAutoHide` is set to  `true`, there isn't a problem as auto hide will hide the splash screen after the configured time.

Reply:
ok thanks - I just wanted to understand the error state